clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -1323,6 +1323,23 @@ def GlobalAnnotationValuesAttr : CIR_Attr<"GlobalAnnotationValues",
   let genVerifyDecl = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// CUDAKernelNameAttr
+//===----------------------------------------------------------------------===//
```

Comment:
We should create CUDA's own attribute file, see openCL example: `clang/include/clang/CIR/Dialect/IR/CIROpenCLAttrs.td`.